api.js

```diff
@@ -466,13 +467,75 @@ function validateObservation (obs) {
   }
 }
 
-var VALID_PROPS = ['lon', 'lat', 'attachments', 'tags', 'ref']
-
-// Filter whitelisted props
+// Top-level props that can be modified by the user/client
+var USER_UPDATABLE_PROPS = [
+  'lon',
+  'lat',
+  'attachments',
+  'tags',
+  'ref',
+  'created_at',
```

Comment:
should we allow clients to update `created_at`?

Reply:
No that's an error. I think clients should be able to initially set created_at (because it should be the time that the user starts creating an observation rather than when they save it I think) but they should not be able to modify it. I'll fix.